### ISSUE TITLE
Adding a digital product to cart, shippingCategoryId Error.

### ISSUE THREAD

**Issue by Floriswijgergangs** [pixelandtonic/DigitalProducts#10]
When submitting a digital product to the cart. Craft Commerce searches for a default shippingCategoryId, this is set in Craft Commerce. Getting this error. Not sure if this is Commerce or Digital Products related.
![image](https://cloud.githubusercontent.com/assets/8255580/19349964/2744d012-9156-11e6-9703-d8788b36ede7.png)

**Comment by lukeholder** [comment]
Thanks for reporting, this will be fixed in the next release of Craft Commerce. In the mean-time you can manual patch the fix by making the following changes to the file:

`craft/plugins/commerce/Commerce/Base/Purchasable.php`

on line 82:

change from:

```
    return craft()->commerce_taxCategories->getDefaultTaxCategory()->id;
```

to:

```
    return \Craft\craft()->commerce_taxCategories->getDefaultTaxCategory()->id;
```

and on line 92:

from:

```
    return craft()->commerce_shippingCategories->getDefaultShippingCategory()->id;
```

to:

```
    return \Craft\craft()->commerce_shippingCategories->getDefaultShippingCategory()->id;
```

The changes above are also here in a screenshot for reference: http://jmp.sh/sJLH7EE

**Comment by lukeholder** [comment]
Fixed in Commerce 1.2.1325

https://craftcommerce.com/changelog#build1325
